Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -944,6 +944,10 @@ final class DateExtensionsTests: XCTestCase {
         XCTAssertTrue(randomDate.isBetween(sinceDate, toDate, includeBounds: true))
     }
 
+    func testYesterday() {
+        let yesterday = Date().yesterday
+        XCTAssert(yesterday.isInYesterday)
```

Comment:
In the right timezone on the right date at the right time, this test could actually fail since it's checking the calendar date components. I mean, the chances of that happening are virtually zero, but [here's some fun reading for you](https://stackoverflow.com/questions/24088730/why-nsdateformatter-is-returning-null-for-a-19-10-2014-in-a-brazilian-time-zone) 😅 (TL;DR Imagine that you run this test at 00:30 where the clock was changed due to daylight savings and now 24 hours ago was actually 25 hours ago and you've gone back 2 days, or something weird like that. It may not be possible but you never know...)
Anyway, I think it might be better to check the time interval difference.